cmd/minishift/cmd/openshift/service.go

```diff
@@ -50,25 +53,78 @@ var serviceCmd = &cobra.Command{
 			atexit.ExitWithMessage(1, "You must specify the name of the service.")
 		}
 
+		host, err := api.Load(constants.MachineName)
+		if err != nil {
+			atexit.ExitWithMessage(1, err.Error())
+		}
+
+		util.ExitIfNotRunning(host.Driver, constants.MachineName)
+
+		ip, err := host.Driver.GetIP()
+		if err != nil {
+			atexit.ExitWithMessage(1, fmt.Sprintf("Error getting IP: %s", err.Error()))
+		}
+
 		service := args[0]
 
-		url, err := openshift.GetServiceSpec(service, namespace, https)
+		serviceSpec, err := openshift.GetServiceSpec(service, namespace, https)
```

Comment:
@hferentschik I do like to have `GetServiceSpec` even it just return serviceSpec for specific service looks much cleaner then. (Internally it does use GetServiceSpecs only)